mono/metadata/profiler.c

```diff
@@ -743,6 +789,38 @@ mono_profiler_class_loaded (MonoClass *klass, int result)
 	}
 }
 
+void
+mono_profiler_static_field (MonoClassField *field)
```

Comment:
What's the purpose of this event? Isn't it too fine grained? It doesn't easily convey to what's going on here, which is layout.

Can't we get the same effect by having callbacks for instance/static layout events?